### ISSUE TITLE
Trying to run the benchmarks, SNAPSHOT deps require local builds?

### ISSUE THREAD

**Issue by nitsanw** [msteindorfer/criterion#1]

**Comment by msteindorfer** [comment]
Hi Nitsan,

This project and other libraries that I'm contributing are currently undergoing restructuring and consolidation. You're right, at the moment the benchmarking library requires local dependencies to compile (and further, some libraries are not available on Maven Central yet). I estimate that these build problems will be fixed by next week (see also [Mavenize the Rascal Suite](https://github.com/cwi-swat/rascal/milestones/Mavenize%20the%20'Rascal%20Suite')).

In the mean time, if you're interested in benchmarking persistent data structures (of Clojure, Scala, and CHAMP) I can refer you to https://github.com/msteindorfer/oopsla15-artifact, that contains an older snapshot of this library and is self-contained.

Let me know if you need further help in running the benchmarks.

Best regards,
Michael

**Comment by nitsanw** [comment]
This will work. Also, for the time being, a regular (or one off) upload of the JMH jar could help. Any chance of that?

**Comment by msteindorfer** [comment]
Yes, there is already an available [JMH jar](https://github.com/msteindorfer/oopsla15-artifact/blob/master/oopsla15-benchmarks/lib/benchmarks.jar) in the [msteindorfer/oopsla15-artifact](https://github.com/msteindorfer/oopsla15-artifact) project. For reproducing results of our paper, the binary should be all you need.

**Comment by msteindorfer** [comment]
Are you planning to reproduce our benchmark results or do you have some extended benchmarks in mind? Anyhow, I would be interested to hear about your results.
